jetty-server/src/main/java/org/eclipse/jetty/server/session/DefaultSessionCache.java

```diff
@@ -37,16 +38,26 @@
     /**
      * The cache of sessions in a hashmap
      */
-    protected ConcurrentHashMap<String, Session> _sessions = new ConcurrentHashMap<>();
+    private final ConcurrentMap<String, Session> _sessions;
 
     private final CounterStatistic _stats = new CounterStatistic();
 
     /**
      * @param manager The SessionHandler related to this SessionCache
      */
     public DefaultSessionCache(SessionHandler manager)
+    {
+        this(manager, new ConcurrentHashMap<>());
+    }
+
+    /**
+     * @param manager The SessionHandler related to this SessionCache
+     * @param sessionMap The session map implementation to use
+     */
+    public DefaultSessionCache(SessionHandler manager, ConcurrentMap<String, Session> sessionMap)
     {
         super(manager);
+        this._sessions = sessionMap;
```

Comment:
```suggestion
        _sessions = sessionMap;
```

Reply:
so whilst we are picking nits... the field and argument names should both be the same.  either `_sessions = sessions` or `_sessionMap = sessionMap`.    Since the field is protected, it is kind of part of the class contract, so I guess it can't be renamed.